sdk/tables/azure-data-tables/azure/data/tables/_table_batch.py

```diff
@@ -15,18 +14,15 @@
 )
 
 from azure.core import MatchConditions
+from azure.core.pipeline.transport import HttpRequest
 
 from ._common_conversion import _transform_patch_to_cosmos_post
 from ._models import UpdateMode, TransactionOperation
 from ._serialize import _get_match_headers, _add_entity_properties, _prepare_key
 from ._entity import TableEntity
-
-if TYPE_CHECKING:
-    from azure.core.pipeline.transport import HttpRequest
-    from ._generated import models, AzureTable
-    from ._generated._configuration import AzureTableConfiguration
-    from ._generated._serialization import Serializer
-    from ._generated._serialization import Deserializer
```

Comment:
Is there a reason you moved all these out of `TYPE_CHECKING`?

Reply:
* Python3 has improved the effiency of import, not worth to have `TYPE_CHECKING`
* Mypy cannot detect the types import under `TYPE_CHECKING`, have to quoted these types in hints

Based on these two reasons, I decided to remove all `TYPE_CHECKING` checks in this package.

Reply:
Sounds good.